glean/src/platform/browser/uploader.ts

```diff
@@ -9,14 +9,16 @@ import { DEFAULT_UPLOAD_TIMEOUT_MS, UploadResultStatus, UploadResult } from "../
 const LOG_TAG = "platform.browser.Uploader";
 
 class BrowserUploader extends Uploader {
+  defaultTimeoutMs: number = DEFAULT_UPLOAD_TIMEOUT_MS;
```

Comment:
should this just be `timeoutMs`?
It's not much of a default anymore if it is changed.

```suggestion
  timeoutMs: number = DEFAULT_UPLOAD_TIMEOUT_MS;
```

Reply:
Good catch :)